HELP_US.txt

```diff
@@ -1,4 +1,4 @@
 We need your help in order to make pH7CMS "the best dating site/app builder" ever made! 
-Thanks for helping us to contribute to this project by adding your code modifications on the official Git repo: https://github.com/pH7Software/pH7-Social-Dating-CMS
+Thanks for helping us to contribute to this project by adding your code modifications on the official Git repositaries: https://github.com/pH7Software/pH7-Social-Dating-CMS
```

Comment:
There is just one repository (URL) here, and "repo" is a common abbreviation for repository in  version-control system words.